controller/accounts.go

```diff
@@ -41,103 +41,43 @@ func NewAccounts(
 	}
 }
 
-func (a *Accounts) GetByID(ID uuid.UUID, projectID uuid.UUID) (*model.Account, error) {
-	var acc model.Account
-
-	err := a.store.GetAccount(ID, projectID, &acc)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to get account")
-	}
-
-	account, err := a.blockchain.GetAccount(projectID, acc.Address)
+func (a *Accounts) GetByAddress(address model.Address, projectID uuid.UUID) (*model.Account, error) {
+	account, err := a.blockchain.GetAccount(projectID, address)
 	if err != nil {
 		return nil, err
 	}
-
-	return account.
-		MergeFromStore(&acc).
-		Export(), nil
+	return account.Export(), nil
 }
 
 func (a *Accounts) AllForProjectID(projectID uuid.UUID) ([]*model.Account, error) {
-	var accounts []*model.Account
-
-	err := a.store.GetAccountsForProject(projectID, &accounts)
+	var proj model.Project
+	err := a.store.GetProject(projectID, &proj)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to get accounts")
+		return nil, err
```

Comment:
it's good to wrap the error with a more understandable message: `failed to get all project accounts`

Reply:
Done